README.md

```diff
@@ -138,6 +138,39 @@ clustering in the `setup_clustering` recipe:
   of the cluster peer member. Only valid when `node['splunk']['clustering']['mode']='slave'`.
   Defaults to 9887.
 
+The following attributes are related to setting up a Splunk server with search head
+clustering in the `setup_shclustering` recipe:
+
+* `node['splunk']['shclustering']`: A hash of search head clustering configurations
+  used in the `setup_shclustering` recipe
+* `node['splunk']['shclustering']['enabled']`: Whether to enable search head clustering,
+  must be set to `true` to use the `setup_shclustering` recipe. Defaults to `false`,
+  must be a boolean literal `true` or `false`.
+* `node['splunk']['shclustering']['mode']`: The search head clustering mode of the node within
+  the cluster. This is used to determine if the node needs to bootstrap the shcluster and begin
+  captain election. Must be set using string literal 'member' or 'captain'.
+* `node['splunk']['shclustering']['label']`: The label for the shcluster. Used to differentiate 
+  from other shclusters in the environment. Must be a string. Defaults to `shcluster1`.
+  captain election. Must be set using string literal 'member' or 'captain'.
+* `node['splunk']['shclustering']['replication_factor']`: The replication factor
+  of the shcluster. Defaults to 2, must be a positive integer.
+* `node['splunk']['shclustering']['replication_port']`: The replication port
+  of the shcluster members. Defaults to 9900.
+* `node['splunk']['shclustering']['deployer_url']`: The management url for the
+  shcluster deployer server, must be set to a string such as: `https://deployer.domain.tld:8089`.
+  Defaults to empty.
+* `node['splunk']['shclustering']['mgmt_uri']`: The management url for the
+  shcluster member node, must be set to a string such as: `https://shx.domain.tld:8089`. You can
+  use the node's IP address instead of the FQDN if desired. Defaults to `https://#{node['fqdn']}:8089`.
+* `node['splunk']['shclustering']['shcluster_members']`: An array of all search head 
+  cluster members referenced by their `mgmt_uri`. Currently this will do a Chef search for nodes that
+  are in the same environment, with search head clustering enabled, and with the same
+  cluster label. Alternatively, this can be hard-coded with a list of all shcluster 
+  members including the current node. Must be an array of strings. Defaults to an empty array.
```

Comment:
I like how you've added support to manually set the list which is useful for environment provisioning based on chef local